fern/pages/sdks/introduction/what-is-an-sdk.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: What is an SDK?
+description: Generate robust, flexible, and automated SDKs in multiple programming languages.
+---
+
+{/* An SDK (Software Development Kit) is any platform-specific tools that 
+enables faster software development. Traditionally, that could include 
+anything from a compiler to documentation to code libraries. */}
+
+SDKs (Software Development Kits) are code libraries that provide functionality for interacting with your API. 
+By simplifying the process of making API calls, developers have a faster time-to-integration
+and run into fewer errors. 
+
+### SDKs vs. vanilla code
+
+Calling an API without an SDK can be tedious and error-prone. An SDK simplifies the headache-inducing parts of 
+calling APIs in code: configuring headers, extracting response data, error-handling, pagination, and rate limits. 
+
+As an example, compare vanilla TypeScript with the Fern SDK to convert text to speech via the ElevenLabs API: 

```

Comment:
the term "Fern SDK" feels ambiguous to me. 

compare vanilla TypeScript (i.e., using `fetch` to make an HTTP call) with a TypeScript SDK generated by Fern ...

Reply:
could also say "built with Fern" if you want to lean into that branding